### PR TITLE
Add Show HN command to hacker-news extension

### PR DESCRIPTION
**Pull request by Yukaii**
## Description

- Add Show HN command to hacker-news extension
- Some code refatoring

## Type of change

- Bug fix / improvement for somebody else's extension. Extension author: @thomaspaulmann 

## Screencast

スクリーンショット 2022-01-07 午後0 16 57

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Review by grzegorzkrukowski — COMMENTED** [review]
Its @thomaspaulmann extension. I will redirect review request to him - always check package.json for original author - I was just probably fixing some issues related to CI

**Review by thomaspaulmann — APPROVED** [review]
Thanks a lot for doing these. Amazing to see more of HN in Raycast :-) All looks good to me, the only "concern" I have is the inconsistent command names: "Read Frontpage - Hacker News" vs. "Show HN - Hacker News". Any thoughts on making this more consistent? 

Maybe something like "Read Show Category - Hacker News". I don't like it that much...

Let me know if you have a better idea and we should address it before merging. Otherwise it should be good to go.

**Comment by Yukaii** [comment]
Thanks for the review!

I'm also struggling when naming the command. Or "Read Show HN" is better?

File: extensions/hacker-news/package.json [hunk 1]
```diff
@@ -13,6 +13,13 @@
       "subtitle": "Hacker News",
       "description": "Get the latest from the fontpage of Hacker News.",
       "mode": "view"
+    },
+    {
+      "name": "show",
+      "title": "Show HN",
```
Comment (thomaspaulmann) [inline extensions/hacker-news/package.json:19]:
```suggestion
      "title": "Read Show HN",
```

**Comment by thomaspaulmann** [comment]
> Thanks for the review!
> 
> I'm also struggling when naming the command. Or "Read Show HN" is better?

Let's go with this...

**Pull request merged**